scripts/package.ps1

```diff
@@ -29,9 +29,17 @@ foreach ($file in [System.IO.Directory]::EnumerateFiles("$repoRoot\src", "System
     }
 }
 
+Ensure-Nuget-Exists
+Write-Host "** Creating BrotliNative NuGet packages. **"
+$brotliExternalFile = "$repoRoot\external\BrotliNative.nuspec"
+Invoke-Expression "$nugetPath pack $brotliExternalFile -Version 0.0.1 -o $packagesPath"
```

Comment:
would it be possible to change package.ps1 to support any project that has a checked-in nuspec file (versus the auto-gen could that exists today)?